src/status_im2/subs/general.cljs

```diff
@@ -12,6 +13,18 @@
  (fn [visibility-status-updates [_ public-key]]
    (get visibility-status-updates public-key)))
 
+(re-frame/reg-sub
+ :visibility-status-updates/online?
+ :<- [:multiaccount/public-key]
+ (fn [my-public-key [_ public-key]]
+   (let [visibility-status-update
+         (if (= public-key my-public-key)
+           @(re-frame/subscribe [:multiaccount/current-user-visibility-status])
```

Comment:
hi, please can you check this commit, if this was right way: https://github.com/status-im/status-mobile/commit/0dfba06869b7e59e62d2ae0a3355cb5255ce3445

Reply:
I deleted it because changes users current visibility status update was affecting(calculating again) subscriptions for all other users.

Reply:
for me it looks like we need to add current-user-visibility-status to :visibility-status-updates/visibility-status-update

Reply:
I pushed commit again for confirmation

Reply:
> I deleted it because changes in users current visibility status update was affecting(calculating again) subscriptions for all other users.

yes that sounds reasonable, but im not sure this might happen often in real life you need to have chat opened with lots of avatars, and change your status , so doesn't sound like a real-life case

Reply:
> for me it looks like we need to add current-user-visibility-status to :visibility-status-updates/visibility-status-update

yes, that's better

Reply:
> > for me it looks like we need to add current-user-visibility-status to :visibility-status-updates/visibility-status-update
> 
> yes, that's better

done, thank you